### ISSUE TITLE
image links not found on commons

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#309]
When building en.wikiversity.org from source, I elected to get the build to download the image files (Thumbnails) as well.
This completed successfully and in the err.txt log file where about 43 lines of the form:

    failed: ttl=[[File:A_Fathers_Love_(10447279325).jpg|220px]]
Different filenames and thumbnail sizes
(There are about 33 unique files)

This means that for this dump these files will always be 'bad' but when accessing a page that has these images, xowa will attempt to download it again (and again)
I think it would be better to have a mechanism to remember this 'badness' and instead show the link in the form wikipedia does (as a red link). [possible as some record in the file-core database]
And for that matter, if a subsequent link is found to be 'bad' it gets added to the blacklist
What do you think?

**Comment by gnosygnu** [comment]
Yeah, I agree this isn't optimal. I considered keeping a "failed-download" list, but I decided against because of:

* Error-prone procedure: File-download was an error-prone procedure, and I didn't want to mark a file bad just because something failed in the download
* Not frequent: This is more of an assumption here, but in most cases, 95% of files on the wiki page do exist; I didn't want to spend time on a feature that would only help for 5% (and have no obvious user effect)
* New Options page: If XOWA had a failed-download feature, then it would need a way to show the failed files as well as clear them (selectively / en masse)

Anyway, let me know if you think differently, and I'll bump this up in the priority queue.
